src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/AbstractCommodity.java

```diff
@@ -0,0 +1,47 @@
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
+
+/**
+ * The class AbstractCommodity represents set of commodities in each room
+ */
+abstract public class AbstractCommodity {
+
+    private static int COUNTER;
+    protected int globalInventoryCounter;
```

Comment:
Sorry my mistake, I meant to say that the `inventoryNumber` should be specific for each object created. and the GLOBAL_INVENTORY_COUNTER should be the static variable which keeps track of all of the instances.